test/spec/modules/quantcastBidAdapter_spec.js

```diff
@@ -125,7 +159,53 @@ describe('Quantcast adapter', function () {
         prebidJsVersion: '$prebid.version$'
       };
 
-      expect(requests[0].data).to.equal(JSON.stringify(expectedBidRequest));
+      expect(requests[0].data).to.equal(JSON.stringify(expectedBannerBidRequest));
+    });
+
+    it('sends video bid requests containing all the required parameters', function () {
+      setupVideoBidRequest();
+
+      const referrer = utils.getTopWindowUrl();
```

Comment:
utils.getTopWindowUrl(); and utils.getTopWindowLocation(); will be removed soon. For this test you can skip site from expectedVideoBidRequest and do assertion.

Reply:
But I use these functions in buildRequests as well, so if I remove site from expectedVideoBidRequest the assertion will fail because site is set in buildRequests.

Or maybe I'm missing something. If that's the case could you, please, give a bit more detail?

Reply:
Can you update your adapter to use http://prebid.org/dev-docs/bidder-adaptor.html#referrers

This new object has everything you need which these functions used to do.

Reply:
I hope it's ok now, though I'm not sure - I don't usually work with JavaScript :)